app/javascript/app/pages/sign-up/sign-up-component.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'redux-first-router-link';
+import { LOGIN } from 'router';
+import cx from 'classnames';
+import Icon from 'components/icon';
+import Input from 'components/input';
+import Button from 'components/button';
+import cwLogo from 'assets/cw-logo.svg';
+import yellowButtonTheme from 'styles/themes/button/button-yellow.scss';
+import styles from './sign-up-styles.scss';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class SignUp extends Component {
+  render() {
+    const { handleValueChange, handleSubmit, fields, errors } = this.props;
+    const hasErrors = errors && errors.length > 0;
+    return (
+      <div className={cx(styles.wrapper, { [styles.withAlert]: alert })}>
+        <div className={styles.logoContainer}>
+          <Icon className={styles.logo} icon={cwLogo} />
+          <div className={styles.logoText}>NDC IMPLEMENTATION TRACKER</div>
+        </div>
+        {fields.map(i => (
+          <Input
+            key={i.slug}
+            label={i.label}
+            inputType={i.type}
+            placeholder={i.placeholder}
+            onChange={value => handleValueChange(i.slug, value)}
+            onBlur={value => handleValueChange(i.slug, value)}
+          />
+        ))}
+        <Button onClick={handleSubmit} theme={yellowButtonTheme}>
```

Comment:
don't we have already the props in Button accepting color? like `<Button color={'yellow'} onClick...>`

Reply:
That is in CW. I prefer the theme approach. Also sometimes there are more changes apart from the color